client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-import-or-migrate/index.tsx

```diff
@@ -16,7 +16,11 @@ import FlowCard from '../components/flow-card';
 import type { Step } from '../../types';
 import './style.scss';
 
-const SiteMigrationImportOrMigrate: Step = function ( { navigation } ) {
+const SiteMigrationImportOrMigrate: Step< {
+	submits: {
+		destination: 'migrate' | 'import' | 'upgrade' | string;
```

Comment:
Maybe we can do
```suggestion
		destination: 'migrate' | 'import' | 'upgrade';
```

Reply:
Based on the code, this more restrictive case should work. These are the only three values we can send in.

Reply:
Fixed.